### PR TITLE
Prevent type error in registerStore

### PR DESCRIPTION
**Pull request by waeltken**
This should fix

    [TypeError: Cannot read property 'instance' of undefined]

in the `registerStore` function.

The added tinytest needs to fail on that type error though. It is currently only visible on the console running the test and won't fail the actual scenario.

### CODE REVIEW

File: test/PgSubscription.js [hunk 1]
```diff
@@ -298,6 +298,16 @@ function(test, done){
...
       test.equal(players.length, expectedRows.length);
+
+      // Test multiple quick changes to subscription
+      for (var i = 0; i < expectedRows.length; i++) {
+        players.change(i);
+      }
+      players.change();
+
+      Meteor.setTimeout(function() {
+        test.equal(player.length, expectedRows.length);
```
Comment (numtel) [inline test/PgSubscription.js]: There is no variable `player`

File: lib/PgSubscription.js [hunk 2]
```diff
@@ -64,6 +64,11 @@ var registerStore = function(connection, name){
       var subBuffer = _.filter(buffer, function(sub){
         return sub.subscriptionId === msg.id;
       })[0];
+
+      if (!subBuffer){
+        return;
```
Comment (numtel) [inline lib/PgSubscription.js]: Why is this the best way to fix this problem?

File: test/PgSubscription.js [hunk 3]
```diff
@@ -298,6 +298,16 @@ function(test, done){
 
     Meteor.setTimeout(function() {
       test.equal(players.length, expectedRows.length);
+
+      // Test multiple quick changes to subscription
```
Comment (numtel) [inline test/PgSubscription.js]: Can you make a separate test case? It makes it much easier to debug if there is a basic change case and then a separate "quick change" test case.

Comment (waeltken) [inline test/PgSubscription.js]: I'll do that. And also the error occurs on the `registerStore`-method so I'll take a look if I can create an actual failing test.

**Pull request closed without merging**